src/navigation/sidebar.js

```diff
@@ -72,7 +72,7 @@ const SideBar = () => {
       items={sideNav}
       style={{ width: "192px" }}
       renderItem={({ href, children }) => {
```

Comment:
Is this working? It shouldn't be. You're removing the `return` then you need an inline body like so:

```js
renderItem={({ href, children }) => <CustomLink to={href}>{children}</CustomLink>}
```

Or

```js
renderItem={({ href, children }) => (<CustomLink to={href}>{children}</CustomLink>)}
```

Reply:
Fixed